resources/authgear/templates/en/web/authflowv2/__authflow_branch.html

```diff
@@ -0,0 +1,152 @@
+{{- define "authflowv2/__authflow_branch.html" }}
+
+{{- if (len $.Branches) }}
+{{ template "authflowv2/__divider.html" . }}
+{{- end }}
+
+<div class="flex flex-col gap-y-4">
+
+  <!-- TODO: Update authenticate with recovery_code to v2 -->
+{{- range $.Branches }}
+{{- if eq $.ActionType "authenticate" }}
+{{- if eq .Authentication "recovery_code" }}
+<form class="text-left text-sm" method="post" novalidate>
+{{ $.CSRFField }}
+<input type="hidden" name="x_index" value="{{ .Index }}">
+<input type="hidden" name="x_channel" value="{{ .Channel }}">
+<input type="hidden" name="x_action" value="take_branch">
+{{ if ($.Translations.HasKey "customer-support-link") }}
+    {{ template "enter-recovery-code-instead-with-customer-support-v1" (dict "customerSupportLink" ($.Translations.RenderText "customer-support-link" nil)) }}
+{{ else }}
+    {{ template "enter-recovery-code-instead-v1" }}
+{{ end }}
+</form>
+{{- end }}
+{{- end }}
```

Comment:
Increase one indentation level for each if / for block if they are not on the same line, or else it could be difficult to read.